### ISSUE TITLE
About velocity contours from the solution domain and local velocity calculated from BEM

### ISSUE THREAD

**Issue by hcOnel** [turbinesFoam/turbinesFoam#262]
Say; for a simulation in which V_inf = 11.4 m/s and TSR = 7, linear speed is roughly 80 m/s and relative velocity is 81 m/s at the blade tip. My question is: should one expect to see this speed after the solution of the flow field in the velocity magnitude contour plot? The maximum velocity I get for such a simulation is around 12 m/s. If this is normal, why? Is it because of the cell size?

**Comment by petebachant** [comment]
I don't think so, since we don't enforce boundary conditions at the blades--only forces. The ALM is only really useful for studying wakes, not necessarily the local blade flow. For that the blade geometry should be resolved in the mesh. Even then, flow at the blade surface will move with the blade due to the no slip condition, but will drop off quickly in the boundary layer.

**Comment by hcOnel** [comment]
The relative velocity of each blade element almost exactly matches with BEM calculations; so if the flow field does not return those exact values, are they calculated in the code manually as a function of r, V_inf, rotational speed and axial induction factors? If so, how is it done for the turbine behind another one?

**Comment by petebachant** [comment]
Relative velocity is calculated from sampling the flow field and adding with the blade element velocity, so it is not manual. The "induction" is handled by Navier--Stokes.
